src/components/Batch/BatchDetails.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import user from "../../assets/purna.webp";
+
+const BatchDetails = () => {
+  const [batchDetails, setBatchDetails] = useState(null);
+  const { id } = useParams();
+
+  useEffect(() => {
+    // Mock data for demonstration
+    const batchData = {
+      2022: {
+        year: 2022,
+        details: "Welcome to the LSPP Batch of 2022!",
+        // members: [
+        //   {
+        //     id: 1,
+        //     name: "Purna Shrestha",
+        //     image: user,
+        //     linkedIn: "https://www.linkedin.com/in/purna-shrestha-0310ps/",
+        //     portfolio: "https://www.purnashrestha.com.np/",
+        //     profile: "",
+        //   },
+        // ],
+        members: Array.from({ length: 20 }, (_, index) => ({
+          id: index + 1,
+          name: "Purna Shrestha",
+          image: user,
+          linkedIn: "https://www.linkedin.com/in/purna-shrestha-0310ps/",
+          portfolio: "https://www.purnashrestha.com.np/",
+          profile: "",
+        })),
+      },
+      2023: {
+        year: 2023,
+        details: "Details for batch 2023...",
+        members: Array.from({ length: 20 }, (_, index) => ({
+          id: index + 1,
+          name: "Purna Shrestha",
+          image: user,
+          linkedIn: "https://www.linkedin.com/in/purna-shrestha-0310ps/",
```

Comment:
Maybe we can use the linkedIn icon for this?

Reply:
> Maybe we can use the linkedIn icon for this?

We will use the icons on all pages. Because we are currently working on MVP, we are attempting to keep it very minimal for now.

Reply:
Even though it's a MVP, we can still use the icons. That will make it look sleek.

Reply:
> Even though it's a MVP, we can still use the icons. That will make it look sleek.

For the time being, the goal is to create reusable components. We haven't decided on which icons to use yet. Choosing whether to use SVG, react, or other resources. So, instead of having to install several packages that will never be utilized in the future. For the time being, only use the text.
I hope you're clear Nishant.